.github/workflows/build.yml

```
@@ -1 +1,46 @@
-#    - name: Build and push Docker image
+name: build
+run-name: ${{ github.actor }} is building ${{ github.ref_name }} 🚀
+on: [push]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+    # Checkout repository
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Log in to the Container registry
+      uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+      with:
+        registry: ${{ env.REGISTRY }}
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    - name: Extract metadata (tags, labels) for Docker
+      id: meta
+      uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+      with:
+        images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+
+    - name: Check variables
+      run: |
+        echo 'Current path:'
+        pwd
+        echo 'Tag: ${{ steps.meta.outputs.tags }}'
+        echo 'Label: ${{ steps.meta.labels.tags }}'
+
+    - name: Build and push Docker image
+      uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
+      with:
+        context: python
```

CI失敗してそうなので、やっぱりこれ戻してください🙏
```suggestion
        context: .
```